mParticle-Example/MPKitExample.m

```diff
@@ -1,24 +1,24 @@
 #import "MPKitExample.h"
+#import <Pilgrim/Pilgrim.h>
 
-/* Import your header file here
-*/
-//#if defined(__has_include) && __has_include(<Example/Example.h>)
-//#import <Example/Example.h>
-//#else
-//#import "Example.h"
-//#endif
+NSString *const PILGRIM_SDK_KEY = @"pilgrim_sdk_key";
+NSString *const PILGRIM_SDK_SECRET = @"pilgrim_sdk_secret";
+NSString *const PERSIST_LOGS = @"pilgrim_sdk_persistent_logs";
+NSString *const MPARTICLE_USER_ID = @"mParticleUserId";
 
-@implementation MPKitExample
+@interface MPKitExample() {
+    FSQPPilgrimManager *pilgrimManager;
```

Comment:
hmm is this something they recommend, using ivars over properties? Maybe this is the right way, I'd like to discuss this further though. Also don't forget to mark this as nullable!!!!
Any time you use instance variables in objective-c they should also be prefixed with an underscore, you can also set ivars in the @implementation:
```
@implementation MPKitExample {
   FSQPPilgrimManager *Nullable _pilgrimManager;
}
```

Reply:
ohhh yeah true, sorry I haven't done objective-C in a bit. I changed it to use a property, makes way more sense